MyTasks/Task2_2/СurrencyСonverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿
+Console.WriteLine("Здравствуйте! Введите сумму, которую хотите конвертировать. Например: 1000");
+double moneyAmount;
+moneyAmount = double.Parse(Console.ReadLine());
+
+Console.WriteLine("Выберите валюту конвертации: USD, EUR, RUB, CAD");
+var currencyName = Console.ReadLine();
+double sum;
```

Comment:
It does not fit the task. Need a transformation, not a validation. Read the task
Dont read it as double we should read '1000 USD' as for example and then convert usd into -> eur, rub, byn,..

Reply:
Do you mean transformation of "one thousand" to "1000"? In this case, I didn't perform this feature as it was an extra task, not obligatory and beyond my abilities, unfortunately.
As to the double, I changed the logic. Now user input is read as "1000 USD" and then converted to the target currency.